llama_index/graph_stores/nebulagraph.py

```diff
@@ -379,19 +379,19 @@ def upsert_triplet(self, subj: str, rel: str, obj: str) -> None:
         # thus we have to assume subj to be the first entity.tag_name
 
         # lower case subj, rel, obj
-        subj = escape_str(subj.lower())
-        rel = escape_str(rel.lower())
-        obj = escape_str(obj.lower())
+        subj = escape_str(subj)
```

Comment:
No, it's case sensitive then, but I aligned the build and query time to be both case sensitive, one hand, it differentiates some words like Apple vs Apple, the other hand I think the ultimate/production level solution should be entity-level embeding/symmetric search based(rather than key word) later I will try adding.

What do you think?

Reply:
This makes sense!